modules/gitops-argo-cd-properties.adoc

```diff
@@ -21,6 +21,8 @@ a|* `<image>` - The container image for the `applicationSet` controller. This ov
   * `<logLevel>` - The log level used by the Argo CD Application Controller component. Valid options are `debug`, `info`, `error`, and `warn`.
   * `<logFormat>` - The log format used by the Argo CD Application Controller component. Valid options are `text` or `json`.
   * `<parallelismLimit>` - The kubectl parallelism limit to set for the controller (`--kubectl-parallelism-limit` flag).
+  * `<sourceNamespaces>` - The list of non-control plane namespaces for creating and managing Argo CD `ApplicationSet` resources in target namespaces.
+  * `<scmProviders>` - The list of URLs of the allowed Source Code Manager (SCM) Providers.
```

Comment:
Not specific to these 2 new additions, but should any of these properties be in `<>`? Are they actually user-replaceable values? They seem specifically pascal cased (e.g., `sourceNamespaces`, `scmProviders`), so maybe all of them in this table (and any other similar like this) should drop the angle brackets?

Reply:
thanks @adellape. We already have a [JIRA ticket ](https://issues.redhat.com/browse/RHDEVDOCS-5940)to focus on them. I will address it as part of that issue.